cmd/root.go

```diff
@@ -28,18 +27,22 @@ var rootCmd = &cobra.Command{
 
 		log.Info("⚡⚡⚡ Starting WattsUp ⚡⚡⚡")
 
-		pool, err := pgxpool.New(context.Background(), config.Database)
+		pool, err := pgxpool.New(ctx, config.Database)
 		if err != nil {
 			log.WithError(err).Error("Failed to create database pool")
 			os.Exit(1)
 		}
```

Comment:
```suggestion
		}
		defer pool.Close()
```

We should add this deferred Close back